src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -367,7 +395,28 @@ def load_command_table(self, _):
         g.custom_show_command('show', 'get_template_spec', validator=_validate_template_spec)
         g.custom_command('list', 'list_template_specs')
         g.custom_command('delete', 'delete_template_spec', validator=_validate_template_spec, confirmation=True)
-
+    
+    with self.command_group('stack mg', resource_deploymentstacks_sdk, resource_type=ResourceType.MGMT_RESOURCE_DEPLOYMENTSTACKS) as g:
+        g.custom_command('show', 'show_deployment_stack_at_management_group', table_transformer=transform_stacks)
+        g.custom_command('list', 'list_deployment_stack_at_management_group', table_transformer=transform_stacks_list)
+        g.custom_command('delete', 'delete_deployment_stack_at_management_group')
```

Comment:
Do we need to consider adding `confirmation=True` for the delete command?

Reply:
Is there a way of having a custom _confirmation_? We were hoping to have the confirmation be a bit more verbose.

Currently, before deleting a stack the following confirmation is prompted:
_Are you sure you want to delete this stack? (y/n):_

Instead of: 
Are you sure you want to perform this operation? (y/n): y

Reply:
Yes, you can use `confirmation='Are you sure you want to delete this stack?'` instead of `confirmation=True`